jina/helper.py

```diff
@@ -188,22 +188,23 @@ def parse_arg(v: str) -> Union[bool, int, str]:
         else:
             return []
     try:
-        v = int(v)  # parse int parameter
+        return int(v)  # parse int parameter
```

Comment:
I prefer to keep a single return if possible (it can make much easier for debugging), plus now return in the end is never reached

Reply:
The return at the end would be reached if both exceptions are raised, and each of the if clauses falls through (i.e. v is not an int, not a float, not length-0, and neither "true" nor "false")